pkg/node/node.go

```diff
@@ -27,9 +27,20 @@ import (
 
 var logger = log.Logger("keep-ecdsa")
 
-const monitorKeepPublicKeySubmissionTimeout = 30 * time.Minute
-const retryDelay = 1 * time.Second
-const blockConfirmations = uint64(12)
+const (
+	// Determines the delay which should be preserved before retrying
+	// actions within the signing process.
```

Comment:
It is also used in the key generation process, so I think: `the signing process.` -> `the key generation and signing process.`